inst/shiny/modules/tab_data/data_mapping.R

```diff
@@ -216,7 +216,9 @@ data_mapping_server <- function(id, adnca_data) {
 
     # Loop through each label and create the renderText outputs
     purrr::walk(MAPPING_DESIRED_ORDER, \(label) {
-      output[[paste0("label_", label)]] <- renderText(get_label(LABELS, label, "ADPC"))
+      output[[paste0("label_", label)]] <- renderText(
+        get_label(labels_df = metadata_nca_variables, label, "ADPC")
```

Comment:
nitpick: No need to provide the default variable. Also, if function has argument names specified, please do try to keep the order the same as in the function definition (`labels_df` is the third argument, here you provide it as first).
```suggestion
        get_label(label, "ADPC")
```